### PR TITLE
update(HK API functions)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

Update the Housekeeper wrapper to reflect changes in the housekeeper package in the following PR.

https://github.com/Clinical-Genomics/housekeeper/pull/111

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/apps/housekeeper/hk.py [hunk 1]
```diff
@@ -274,11 +274,11 @@ def get_create_version(self, bundle_name: str) -> Version:
 
     def new_tag(self, name: str, category: str = None):
         """Create a new tag."""
-        return self._store.new_tag(name, category)
+        return self._store.create_tag(name, category)
 
     def add_tag(self, name: str, category: str = None) -> models.Tag:
```
Comment (henrikstranneheim) [inline cg/apps/housekeeper/hk.py:279]:
```suggestion
    def add_tag(self, name: str, category: str = None) -> Tag:
```

File: cg/apps/housekeeper/hk.py [hunk 2]
```diff
@@ -274,11 +274,11 @@ def get_create_version(self, bundle_name: str) -> Version:
...
         """Add a tag to the database."""
-        tag_obj = self._store.new_tag(name, category)
+        tag_obj = self._store.create_tag(name, category)
```
Comment (henrikstranneheim) [inline cg/apps/housekeeper/hk.py:281]:
```suggestion
        tag: Tag = self._store.create_tag(name, category)
```
Use named args

**Review by diitaz93 — APPROVED** [review]
👍

**Review by henrikstranneheim — APPROVED** [review]
👍

**Review by seallard — COMMENTED** [review]
- add_file does not exist any longer, right?

**Pull request closed without merging**